climakitae/core/constants.py

```diff
@@ -0,0 +1,4 @@
+# constants.py
+"""This module defines constants across the codebase"""
+
+WARMING_LEVELS = [0.8, 1.2, 1.5, 2.0, 2.5, 3.0, 4.0]
```

Comment:
I really like the idea of a `constants.py` file, however, I think it's implementation at this stage is incomplete. Doing a search across climakitae, you can see that the string "1.5, 2.0", for example, is found in 11 files: https://github.com/search?q=repo%3Acal-adapt%2Fclimakitae%201.5%2C%202.0&type=code

To properly make use of a `constants.py` file, we should be replacing anywhere the warming levels list is used with these constants, like you've done in `data_interface.py` here. Which, you could do in this PR if you want, or you could remove constants.py here and do that entire change in a separate PR (which I am more in favor of, personally). 

I'm not sure if we need to be actively updating some of the files from that link with the newest things. For example, I'm not sure if we need to be updating `vulnerability.py`-- perhaps check in with Mark about that? Same with `uncertainty.py`. I don't know what [this file]([climakitae/data/gwl_1981-2010ref_EC-Earth3_ssp370.csv](https://github.com/cal-adapt/climakitae/blob/afc2dc2618240a85d31cab51c80dd9b7b7b0daf5/climakitae/data/gwl_1981-2010ref_EC-Earth3_ssp370.csv)) is but it needs (?) to be updated as well.

**TLDR**: If we don't want to update anything but `data_interface.py`, I don't think it makes sense to have a `constants.py` file, and I think it overcomplicates the code. However, if we want to update all the other files that use warming levels, the `constants.py` file should be read in and used all those files as well (i.e. replace all the strings).

Reply:
Yeah, I just replaced it in one place for now to show how it could be used and see people's thoughts before replacing it all the other places too. 

If we do go forward with constants.py, I think that we should replace all references of warming levels with this constant, and within specific files that may not use all warming levels (like `vulnerability.py` or `uncertainty.py`), we can modify the constant in-place within that file so that it's clearly demonstrated that _specific_ warming levels were extracted from all the potential warming levels in our codebase.

Reply:
I can also separate a `constants.py` into a different PR and get this one across the finish line without that first.